triplets/community/arm64-osx-dynamic-release.cmake

```diff
@@ -6,3 +6,6 @@ set(VCPKG_CMAKE_SYSTEM_NAME Darwin)
 set(VCPKG_OSX_ARCHITECTURES arm64)
 
 set(VCPKG_BUILD_TYPE release)
+
+set(VCPKG_C_FLAGS -mmacosx-version-min=11.0)
+set(VCPKG_CXX_FLAGS -mmacosx-version-min=11.0)
```

Comment:
Do we also need to specify this in the triplets?

```
set(VCPKG_LINKER_FLAGS -mmacosx-version-min=11.0)
```

I was looking at these:
- https://stackoverflow.com/questions/63099524/how-to-set-macosx-deployment-target-in-meson-for-pango-harfbuzz-glib-etc
- https://cmake.org/cmake/help/latest/envvar/LDFLAGS.html
- https://learn.microsoft.com/en-us/vcpkg/users/triplets#vcpkg_linker_flags

Reply:
Looks like you might be right: https://stackoverflow.com/a/3899165

I'll add that to be safe.

Reply:
I made the changes and ran the tests again to verify that the minos is still 11.0.